packages/components/pie-notification/test/visual/pie-notification.spec.ts

```diff
@@ -120,201 +42,204 @@ test.describe('Props', () => {
 
-            test('should not stack buttons when hasStackedActions is true and isCompact is true regardless the screen size', async ({ mount, page }) => {
+            test('should not stack buttons when hasStackedActions is true and isCompact is true regardless the screen size', async ({ page }) => {
                 // Arrange
-                await mount(PieNotification, {
-                    props: {
-                        ...initialValues,
-                        leadingAction: mainAction,
-                        supportingAction: secondaryAction,
-                        hasStackedActions: true,
-                        isCompact: true,
-                    } as NotificationProps,
-                });
-
+                const basePage = new BasePage(page, 'notification');
+                const props: Partial<NotificationProps> = {
+                    ...initialValues,
+                    leadingAction: mainAction,
+                    supportingAction: secondaryAction,
+                    hasStackedActions: true,
+                    isCompact: true,
+                };
+
+                await basePage.load({ ...props });
+
+                // Assert
                 await percySnapshot(page, 'PieNotification - hasStackedActions = false, isCompact = true - should not stack buttons');
             });
         });
     });
 
     test.describe('PieNotification positions', () => {
         positions.forEach((positionValue) => {
-            test(`should render correctly the ${positionValue} position`, async ({ page, mount }) => {
-                await mount(PieNotification, {
-                    props: {
-                        ...initialValues,
-                        position: positionValue,
-                    } as NotificationProps,
-                    slots: { default: slotContent },
-                });
+            test(`should render correctly the ${positionValue} position`, async ({ page }) => {
+                // Arrange
+                const basePage = new BasePage(page, 'notification');
+                const props: Partial<NotificationProps> = {
+                    ...initialValues,
+                    position: positionValue,
+                };
+
+                await basePage.load({ ...props });
 
+                // Assert
                 await percySnapshot(page, `PieNotification - position = ${positionValue} border-radius`);
             });
         });
     });
 });
 
 test.describe('Reading direction - RTL - Right to Left', () => {
-    test('should slots icons and buttons when the reading direction is RTL', async ({ mount, page }) => {
-        setRTL(page);
-
-        const mainAction = {
-            text: confirmRTL,
-        };
-        const secondaryAction = {
-            text: denyRTL,
+    test('should slots icons and buttons when the reading direction is RTL', async ({ page }) => {
+        // Arrange
+        const basePage = new BasePage(page, 'notification--rtl-visual');
```

Comment:
This will fix the broken test as seen in Percy.

```suggestion
        const basePage = new BasePage(page, 'notification--notification-rtl');
```